### PR TITLE
SB-14: feat: setup MediaTypes feature

### PR DESCRIPTION
**Pull request by eccentricnode**
## Feature/Problem

New NgRx feature, MediaTypes

## Solution

Implemented new MediaTypes NgRx feature

## Areas of Impact

Core-State

## UI Images

N/A

## Manual Testing

- Verify CI passes

### CODE REVIEW

File: client/libs/core-state/src/lib/core-state.module.ts [hunk 1]
```diff
@@ -31,7 +34,12 @@ const storeConfig: RootStoreConfig<any> = {
       TagsEffects,
       MediaEffects
     ]),
-    StoreDevtoolsModule.instrument({ name: 'SermonBeacon Store' })
+    StoreDevtoolsModule.instrument({ name: 'SermonBeacon Store' }),
```
Comment (VicAv99) [inline client/libs/core-state/src/lib/core-state.module.ts]: Why was this changed to forFeature?

File: client/libs/core-state/src/lib/media-types/media-types.actions.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+);
+
+// Create Actions
+export const createMediaType = createAction(
+  '[MediaTypes] Create New MediaType',
+  props<{ mediaType: MediaType }>()
+);
+
+export const createMediaTypeSuccess = createAction(
+  '[MediaTypes] New MediaType Successfully Created',
```
Comment (aklukas) [inline client/libs/core-state/src/lib/media-types/media-types.actions.ts]:
```suggestion
  '[MediaTypes] MediaType Successfully Created',
```

File: client/libs/core-state/src/lib/media-types/media-types.actions.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+export const loadMediaTypes = createAction('[MediaTypes] Load MediaTypes');
+
+export const loadMediaTypesSuccess = createAction(
+  '[MediaTypes] MediaTypes Successfully Loaded',
+  props<{ mediaTypes: MediaType[] }>()
+);
+
+// Create Actions
+export const createMediaType = createAction(
+  '[MediaTypes] Create New MediaType',
```
Comment (aklukas) [inline client/libs/core-state/src/lib/media-types/media-types.actions.ts]:
```suggestion
  '[MediaTypes] Create MediaType',
```
